examples/basics/visuals/polygon_prototype.py

```diff
@@ -0,0 +1,49 @@
+from vispy import app
+import sys
+from vispy.scene import SceneCanvas
+from vispy.scene.visuals import Polygon, Ellipse, Rectangle
+from vispy.color import Color
+
+white = Color("#ecf0f1")
+gray = Color("#121212")
+red = Color("#e74c3c")
+blue = Color("#2980b9")
+orange = Color("#e88834")
```

Comment:
@bollu after you are done with visual development, if you want to start working on making the `vispy.plot` functions have nice defaults it would be worthwhile. You seem to have a good handle on what some good defaults are. Related issues:

https://github.com/vispy/vispy/issues/755
https://github.com/vispy/vispy/issues/742